packages/roosterjs-content-model/lib/publicTypes/format/formatParts/DirectionFormat.ts

```diff
@@ -16,4 +16,9 @@ export type DirectionFormat = {
      * Whether the text align value comes from attribute rather than CSS
      */
     isTextAlignFromAttr?: boolean;
+
+    /**
+     * Horizontal alignment of an element
+     */
+    alignSelf?: string;
```

Comment:
Instead of adding this raw CSS here, can we just respect the single `textAlign` property?

Content Model property doesn't need to 1:1 map to CSS property, but the goal is to be easier to do format, that means, when change alignment, we just need to change single property, then let content processor/handler to handle different CSS properties when create Content Model or write back